api_tests/lib/btsieve.ts

```diff
@@ -0,0 +1,71 @@
+import * as chai from "chai";
+
+import chaiHttp = require("chai-http");
+chai.use(chaiHttp);
+
+import { Transaction } from "web3-core";
+import { TransactionReceipt } from "web3-core";
+
+export interface IdMatch {
+    id: string;
+}
+
+export interface BitcoinResponse {
+    query: any;
+    matches: IdMatch[];
+}
+
+export interface EthereumResponse {
+    query: any;
+    matches: EthereumMatch[];
```

Comment:
:x: This does not accurately represent the API. You can choose the type of response for each ledger so `EthereumResponse` might also have `IdMatch`s, depending on which query parameter you return.

Reply:
I updated, tell me what you think and/or propose alternative.